pom.xml

```diff
@@ -92,11 +98,22 @@
       <name>CheckmarxDev</name>
       <url>https://maven.pkg.github.com/CheckmarxDev/ast-cli-java-wrapper</url>
     </repository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
+<!--    <repository>-->
```

Comment:
I think we can remove the commented code. If we need in the future we can check previous versions.